imports/ui/dialogs/create-report-dialog.jsx

```diff
@@ -25,12 +26,26 @@ class CreateReportDialog extends Component {
       }
     }, 300)
   }
+
+  creationTime () {
+    const creationTime = (new Date()).toISOString()
+    return moment(creationTime).format('DD/MM/YYYY')
```

Comment:
`creationTime` isn't needed.
`moment()` as-is creates a "moment" object that wraps the current date.
Replacing both lines by `return moment().format('DD/MM/YYYY')` would have the same effect